docs/get-details/transactions/transactions.md

```diff
@@ -63,7 +63,7 @@ Object Name: `AssetParams`
 |Field|Required|Type|codec| Description|
 |---|---|---|---|---|
 |<a name="total">Total</a>|_required on creation_|uint64|`"t"`| The total number of base units of the asset to create. This number cannot be changed.|
-|<a name="decimals">Decimals</a>|_required on creation_|uint32|`"dc"`| The number of digits to use after the decimal point when displaying the asset. If 0, the asset is not divisible. If 1, the base unit of the asset is in tenths. If 2, the base unit of the asset is in hundredths. |
+|<a name="decimals">Decimals</a>|_required on creation_|uint32|`"dc"`| The number of digits to use after the decimal point when displaying the asset. If 0, the asset is not divisible. If 1, the base unit of the asset is in tenths. If 2, the base unit of the asset is in hundredths. If 3, then the base unit of the asset is in thousandths, this should go on in an ascending manner until 8 or 9 decimals. |
```

Comment:
```suggestion
|<a name="decimals">Decimals</a>|_required on creation_|uint32|`"dc"`| The number of digits to use after the decimal point when displaying the asset. If 0, the asset is not divisible. If 1, the base unit of the asset is in tenths. If 2, the base unit of the asset is in hundredths. If 3, then the base unit of the asset is in thousandths, and so on up to 19 decimal places |
```

Reply:
@barnjamin @nullun just made this change, thank you.